### ISSUE TITLE
Sending messages to individuals 

### ISSUE THREAD

**Issue by PooriaT** [PooriaT/P2Talk#2]
The current state of code is broadcasting the message to all clients. Each client should have an accessibility of sending message to an individual.

**Comment by PooriaT** [comment]
Solved.
